user-app-test/src/modules/users/dto/create-user.dto.ts

```diff
@@ -0,0 +1,17 @@
+import { IsEmail, IsString } from 'class-validator';
+import { IsAge } from 'src/decorators/is-age.decorator';
+import { IsPassword } from 'src/decorators/is-password.decorator';
+
+export class CreateUserDto {
+  @IsString()
+  name: string;
```

Comment:
```suggestion
  @IsString()
  @IsNotEmpty()
  name: string;
```
иначе `""` тоже прокатит